src/services/RemoveCookies.ts

```diff
@@ -0,0 +1,11 @@
+'use client'
+import { useCookies } from '@/services/CookiesService'
+
+export function handleRemoveCookie() {
+  const { removeCookie } = useCookies()
+
+  removeCookie('token')
+  removeCookie('user')
+  removeCookie('refreshToken')
+  window.location.href = '/login'
```

Comment:
usar [useRouter](https://nextjs.org/docs/app/api-reference/functions/use-router) se for client component, e para server components use o  [redirect](https://nextjs.org/docs/app/api-reference/functions/redirect)